examples/localvideosnapshot/public/index.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto+Mono:300');
```

Comment:
Many of the rules in `examples/localvideosnapshot/public/index.css` should be shared by `examples/mediadevices/public/index.css`. Can you factor these out? Presumably any additional examples will share them, too. I'd like them to share the styles in case we want to re-skin everything later.